internal/activity.go

```diff
@@ -165,6 +165,10 @@ type (
 		//
 		// NOTE: Experimental
 		Summary string
+
+		// Priority - Optional priority and fairness settings that control relative ordering of
+		// task processing when tasks are backed up in a queue.
```

Comment:
The docs on `commonpb.Priority` are already very detailed, no need to duplicate them here.

Reply:
Just a note, I know this is in draft, but we don't reference raw proto types in SDK apis. We generally create a native SDK type and translate

Reply:
See, for example, `RetryPolicy` and `commonpb.RetryPolicy` in these options